<!DOCTYPE html>
<html>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/3/w3.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
<head>
</head>
<body>

<nav class="w3-bar w3-black">
	<a href="index.html" class="w3-button w3-bar-item">HOME</a>
	<a href="research.html" class="w3-button w3-bar-item">RESEARCH</a>
	<a href="people.html" class="w3-button w3-bar-item">PEOPLE</a>
	<a href="software.html" class="w3-button w3-bar-item">SOFTWARE & DATA</a>
	<a href="publications.html" class="w3-button w3-bar-item">PUBLICATIONS</a>
	<a href="misc.html" class="w3-button w3-bar-item">MISCELLANEOUS</a>
	<!-- <a href="openpositions.html" class="w3-button w3-bar-item">OPEN POSITIONS</a> -->
</nav>

<!-- 4. Add main title and subtitle-->

<section class="w3-container w3-center">
	<h1 class="w3-wide">CHAVEZ LABORATORY</h1>
	<!-- <h4 class="w3-opacity w3-wide">CHROMATIN AND GENE REGULATION IN HEALTH AND DISEASE</h4> -->
</section>

<style>

	.clearfix{
		overflow: auto;
	}
	.story{
		max-width: 800px;
		margin: auto;
		background: transparent;
		padding: 10px;
	}

</style>


<div class="clearfix story">
<h3 class="w3-wide">SOFTWARE FROM OUR LAB <i class="fa fa-github" style="font-size: 30px"></i></h3>

<br>

<h4><a href="https://bioconductor.org/packages/release/bioc/html/InTAD.html" target="_blank"><b>InTAD</b> - Chromosomal conformation guided analysis of enhancer target genes</a></h4>

<p class="w3-justify">The package is focused on the detection of correlation between expressed genes and selected epigenomic signals i.e. enhancers obtained from ChIP-seq data within topologically associated domains (TADs). Various parameters can be controlled to investigate the influence of external factors and visualization plots are available for each analysis step.</p>

<p><b>Bioconductor page: <a href="https://bioconductor.org/packages/release/bioc/html/InTAD.html" target="_blank">InTAD</a></b></p>

<br>

<h4><a href="https://bioconductor.org/packages/release/bioc/html/qsea.html" target="_blank"><b>QSEA</b> - <b>Q</b>uantitative <b>S</b>equencing <b>E</b>nrichment <b>A</b>nalysis</a></h4>

<p class="w3-justify">qsea (quantitative sequencing enrichment analysis) was developed as the successor of the MEDIPS package for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, qsea provides several functionalities for the analysis of other kinds of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential enrichment between groups of samples.</p>

<p><b>Bioconductor page: <a href="https://bioconductor.org/packages/release/bioc/html/qsea.html" target="_blank">QSEA</a></b></p>

<br>

<h4> <a href="https://bioconductor.org/packages/release/bioc/html/MEDIPS.html" target="_blank"><b>MEDIPS</b> - <b>ME</b>thylated <b>D</b>NA <b>I</b>mmuno<b>P</b>recipitation <b>S</b>eq analysis</a></h4>
<p class="w3-justify">MEDIPS was developed for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, MEDIPS provides functionalities for the analysis of any kind of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential coverage between groups of samples and saturation and correlation analysis.</p>

<p><b>Bioconductor page: <a href="https://bioconductor.org/packages/release/bioc/html/MEDIPS.html" target="_blank">MEDIPS</a></b></p>

</div>


<!-- Content should be here-->

<footer class="w3-container w3-padding-60 w3-center w3-black w3-large">
	<!-- <p><font size="1.5">Designed by <a href="https://twitter.com/nerd_yie">@nerd_yie</a></font></p> -->
	<p></p>
</footer>

<footer class="w3-container w3-padding-20 w3-center w3-white w3-large">

</footer>

</body>
</html>
